forgit.plugin.zsh

```diff
@@ -38,20 +35,20 @@ forgit::diff() {
     forgit::inside_work_tree || return 1
     local cmd files opts commit repo
     [[ $# -ne 0 ]] && {
-        if $forgit_git_no_pager rev-parse "$1" -- &>/dev/null ; then
+        if git --no-pager rev-parse "$1" -- &>/dev/null ; then
```

Comment:
does it necessary to add `--no-pager` here?

Reply:
Yea no actually. It's my inexperience with tty/pty showing through. Thanks for that quora link, great info!

I've redacted my `--no-pager` changes.